client/localization/extract_static_strings.py

```diff
@@ -8,6 +8,9 @@
 from bs4 import BeautifulSoup
 from bs4.element import NavigableString
 
+AVAILABLE_LANGUAGES = ['en', 'pl']
+AVAILABLE_LANGUAGES.remove('en')
```

Comment:
these two lines don't make sense.

Reply:
not true. This way you don't have to care about which language is 'source' one so you just write list of all available languages